.flake8

```diff
@@ -227,3 +267,42 @@ per-file-ignores =
   tests/unit/image_manager/__init__.py: D104
   tests/unit/__init__.py: D104
   share/ansible_navigator/utils/__init__.py: D104
+
+# Count the number of occurrences of each error/warning code and print a report:
+statistics = true
+
+# flake8-eradicate
+# E800:
+eradicate-whitelist-extend = isort:\s+\w+
+
+# flake8-pytest-style
+# PT001:
+pytest-fixture-no-parentheses = true
+# PT006:
+pytest-parametrize-names-type = tuple
```

Comment:
This is also exempted on a per-file basis, can we have one or the other? The two ways to have the exception seem to conflict, maybe update the per-file exception in another PR?

Reply:
It's not an exception. Some of the flake8 plugins have their own configurations. They are per-rule, not per-file. This setting is just a baseline for how the plugin is supposed to work.

Reply:
got it!

Reply:
Yep, basically, there are two styles. And one is chosen by default making the other produce a violation of the given linter rule. When the setting is changed, the violation is raised in the opposite case.